python/tk_multi_workfiles/actions/interactive_open_action.py

```diff
@@ -236,6 +236,12 @@ def _open_workfile(self, file, env, parent_ui):
                     src_path = work_path
                     work_path = local_path
 
+        try:
+            self._app.log_metric("Open workfile")
```

Comment:
When I said destination, I meant whether users were opening the file in the sandbox or copying it to theirs first and then opening. Not the actual file path.

I agree tough that this might be too fine grained. It would probably be better to track this at the pipeline level if we ever wanted to track it.